src/libexpr/primops.cc

```diff
@@ -2420,12 +2420,14 @@ static RegisterPrimOp primop_listToAttrs({
       Construct a set from a list specifying the names and values of each
       attribute. Each element of the list should be a set consisting of a
       string-valued attribute `name` specifying the name of the attribute,
-      and an attribute `value` specifying its value. Example:
+      and an attribute `value` specifying its value. In case of duplicate
+      occurrences of the same name, the first takes precedence. Example:
```

Comment:
Please add whitespace, if text is too dense it's hard to find the bits of information by scanning.
```suggestion
      and an attribute `value` specifying its value.
      
      In case of duplicate occurrences of the same name, the first takes precedence.
      
      Example:
```

Reply:
Will do.